providers/azure/resources/azure.lr

```diff
@@ -1137,8 +1137,8 @@ private azure.subscription.sqlService.server.administrator @defaults("id name")
   tenantId string
 }
 
-// Azure SQL server database
-private azure.subscription.sqlService.database @defaults("id name") {
+// Azure SQL Database service database
```

Comment:
server, not service?

Reply:
The full name is Azure SQL Database service so using the full name allows us not to have Database database. The naming of this product is problematic.